Gems/Pointcloud/gem.json

```
@@ -1,6 +1,6 @@
 {
     "gem_name": "Pointcloud",
-    "version": "2.0.1",
+    "version": "3.0.1",
```

When bumping major, you should reset patch.
```suggestion
    "version": "3.0.0",
```